R/CoreMethods.R

```diff
@@ -149,7 +149,7 @@ sc3_prepare.SingleCellExperiment <- function(object, gene_filter, pct_dropout_mi
     f_data <- rowData(object)
     f_data$sc3_gene_filter <- TRUE
     if (gene_filter) {
-        dropouts <- rowSums(counts(object) == 0)/ncol(object)*100
+        dropouts <- Matrix::rowSums(counts(object) == 0)/ncol(object)*100
```

Comment:
I am aware of the discussion in #53 , but I am still not sure if hardcoding it this way without checking the type of `counts(object)` is a good idea. For hdf5-backed object, the returned matrix can be of class "DelayedMatrix", which is not handled correctly by Matrix::rowSums(), just as an example.

Reply:
Nice, thanks for the comment! That's why I didn't rush to merge it, the issue is quite big and requires some time to think and decide.